### PR TITLE
add some snapshots and actions tests

### PR DESCRIPTION
**Pull request by NikBorn**
I need you to please review my App snapshot and my Action tests.  I will still need to add some additional tests but I wanted to get this merged in before I add some of the additional functionality that was requested.

### CODE REVIEW

**Comment by NikBorn** [comment]
@nicktu12
@lolakoala

File: src/tests/snapshot-tests/App.test.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import { shallow, mount, configure } from 'enzyme';
+import App from '../../Components/App/App.js';
+import Adapter from 'enzyme-adapter-react-16';
+import configureStore from 'redux-mock-store';
+
+
+configure({ adapter: new Adapter() });
+
+describe('App', () => {
+  // const mockStore = configureStore();
```
Comment (nicktu12) [inline src/tests/snapshot-tests/App.test.js]: sad dead code

File: src/tests/snapshot-tests/LocationList.test.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import { shallow, mount, configure } from 'enzyme';
-import LocationList from '../Components/LocationList/LocationList.js';
+import LocationList from '../../Components/LocationList/LocationList.js';
 import Adapter from 'enzyme-adapter-react-16';
-import { MockLocation, mockLocationsResponse } from '../Utils/mockData.js';
+import { mockLocation, mockLocationsResponse } from '../../Utils/mockData.js';
```
Comment (nicktu12) [inline src/tests/snapshot-tests/LocationList.test.js]: nice rename!

**Comment by nicktu12** [comment]
Snapshot tests are groovy

File: src/tests/snapshot-tests/App.test.js (hunk 1 above)
Comment (NikBorn) [reply src/tests/snapshot-tests/App.test.js]: thanks for the catch!

**Comment by lolakoala** [comment]
Action tests look good. Good job mocking out all that data.

**Pull request merged**